### ISSUE TITLE
[Components and pattern standards] Design components and patterns in use are outdated.  (04.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#60374]
#### VFS team name
Digitize Forms

#### VFS product name
Form 21-4142(a)

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components and patterns in use are outdated. 

### Issue Details
The privacy policy link in the statement of truth section does not indicate to users that the link to review this document is external and will open in a new page. This indicated should be visual and auditory.

### VA.gov Experience Standard
[Category Number 04, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Guidance on this component has been updated. Use the [privacy agreement](https://design.va.gov/storybook/?path=/docs/components-va-privacy-agreement--default) component in Storybook and ensure that you follow behavior guidance from the Design System.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by jeana-adhoc** [comment]
@ndsprinkle On the "Statement of truth" privacy policy link - we need to add 

```
<i class="fa-arrow-up-right-from-square" aria-hidden="true" role="img"></i>   
<span class="sr-only">opens in a new window</span>
```

inside of the privacy policy link

so it ends up like this

```
<a href="/privacy-policy/" target="_blank">   
  privacy policy
  <i class="fa-arrow-up-right-from-square" aria-hidden="true" role="img"></i>
  <span class="sr-only">opens in a new window</span>
</a>
```

This should be done to all statement of truths using our pattern. 

The end result looks like this:

<img width="450" alt="statement of truth" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/de537ef4-7914-4b4e-9b38-cee3af961bf6">

**Comment by ndsprinkle** [comment]
Issue ticket created for design system team: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1886

**Comment by shiragoodman** [comment]
@ndsprinkle sorry, opening this back up. We ask that tickets not be closed until the issue is fixed. Thanks for your understanding.

**Comment by jeana-adhoc** [comment]
Thanks, Shira - we should have documented this better.

The form has been updated with the icon. (Screenshot from production)

<img width="624" alt="Statement of truth with privacy icon" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/117098918/7a97f8ef-b019-4df6-b07d-1452501bffd8">

The ticket for the Design System team is to turn this into a re-usable component. :)

**Comment by humancompanion-usds** [comment]
I've https://design.va.gov/patterns/ask-users-for/signature#examples. I don't think this ticket should stay open until we add an experimental component. That would be an odd Inception problem as this issue is about not matching the pattern but the component doesn't exist yet. The pattern is documented and is codified in the Forms library. Adding the new component to the Design System is orthogonal to this issue.
